.github/workflows/pull_request.yaml

```diff
@@ -85,11 +85,15 @@ jobs:
             restore-keys: |
               ${{ runner.os }}-go
 
+        - name: Set up Docker Buildx
+          uses: docker/setup-buildx-action@v2
+
         # build the openfga/openfga image for the functional tests
         - uses: docker/build-push-action@v3
           with:
             file: Dockerfile
             push: false # don't publish the built container for functional tests
+            load: true
```

Comment:
By default the `buildx` engine will build the container and put it in the image cache, but it won't load it into the local docker registry. `load` ensures that the build image is put into the local docker registry so that the subsequent command `make functional-test` will have it available when it runs `docker pull openfga/openfga:functionaltest`